### PR TITLE
Documentación lineas extra

### PR DESCRIPTION
**Pull request by polsala**
### Description:

> Description of the docs to add/modify

### Tasks:

- [x] Documentar lineas extra
- [x] Añadir esquematico para la facturación de lineas extra

### Links:

> Add links for each file modified

* [lineas_extra #Linies extra](http://builds.gisce.net/powerp-docs/powerp_docul_extra_lines/ca/facturacion/lineas_extra/)
* [lineas_extra #Linies extra](http://builds.gisce.net/powerp-docs/powerp_docul_extra_lines/es/facturacion/lineas_extra/)

### CODE REVIEW

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_docul_extra_lines/

Things to review:
- [ ] Typos in Catalan
- [ ] Typos in Spanish
- [ ] Links routes in Spanish
- [ ] Links work in Catalan
- [ ] Links work in Spanish
- [ ] Images in Spanish
- [ ] Page index shows all headers

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_docul_extra_lines/

Things to review:
- [ ] Typos in Catalan
- [ ] Typos in Spanish
- [ ] Links routes in Spanish
- [ ] Links work in Catalan
- [ ] Links work in Spanish
- [ ] Images in Spanish
- [ ] Page index shows all headers

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_docul_extra_lines/

Things to review:
- [ ] Typos in Catalan
- [ ] Typos in Spanish
- [ ] Links routes in Spanish
- [ ] Links work in Catalan
- [ ] Links work in Spanish
- [ ] Images in Spanish
- [ ] Page index shows all headers

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_docul_extra_lines/

Things to review:
- [ ] Typos in Catalan
- [ ] Typos in Spanish
- [ ] Links routes in Spanish
- [ ] Links work in Catalan
- [ ] Links work in Spanish
- [ ] Images in Spanish
- [ ] Page index shows all headers

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_docul_extra_lines/

Things to review:
- [ ] Typos in Catalan
- [ ] Typos in Spanish
- [ ] Links routes in Spanish
- [ ] Links work in Catalan
- [ ] Links work in Spanish
- [ ] Images in Spanish
- [ ] Page index shows all headers

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_docul_extra_lines/

Things to review:
- [ ] Typos in Catalan
- [ ] Typos in Spanish
- [ ] Links routes in Spanish
- [ ] Links work in Catalan
- [ ] Links work in Spanish
- [ ] Images in Spanish
- [ ] Page index shows all headers

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_docul_extra_lines/

Things to review:
- [ ] Typos in Catalan
- [ ] Typos in Spanish
- [ ] Links routes in Spanish
- [ ] Links work in Catalan
- [ ] Links work in Spanish
- [ ] Images in Spanish
- [ ] Page index shows all headers

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_docul_extra_lines/

Things to review:
- [ ] Typos in Catalan
- [ ] Typos in Spanish
- [ ] Links routes in Spanish
- [ ] Links work in Catalan
- [ ] Links work in Spanish
- [ ] Images in Spanish
- [ ] Page index shows all headers

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_docul_extra_lines/

Things to review:
- [ ] Typos in Catalan
- [ ] Typos in Spanish
- [ ] Links routes in Spanish
- [ ] Links work in Catalan
- [ ] Links work in Spanish
- [ ] Images in Spanish
- [ ] Page index shows all headers

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_docul_extra_lines/

Things to review:
- [ ] Typos in Catalan
- [ ] Typos in Spanish
- [ ] Links routes in Spanish
- [ ] Links work in Catalan
- [ ] Links work in Spanish
- [ ] Images in Spanish
- [ ] Page index shows all headers

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_docul_extra_lines/

Things to review:
- [ ] Typos in Catalan
- [ ] Typos in Spanish
- [ ] Links routes in Spanish
- [ ] Links work in Catalan
- [ ] Links work in Spanish
- [ ] Images in Spanish
- [ ] Page index shows all headers

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_docul_extra_lines/

Things to review:
- [ ] Typos in Catalan
- [ ] Typos in Spanish
- [ ] Links routes in Spanish
- [ ] Links work in Catalan
- [ ] Links work in Spanish
- [ ] Images in Spanish
- [ ] Page index shows all headers

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_docul_extra_lines/

Things to review:
- [ ] Typos in Catalan
- [ ] Typos in Spanish
- [ ] Links routes in Spanish
- [ ] Links work in Catalan
- [ ] Links work in Spanish
- [ ] Images in Spanish
- [ ] Page index shows all headers

File: docs/facturacion/lineas_extra.md [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+Referent als processos de facturació automàtica, la facturació de línies extra
+seguirà les condicions de facturació definides en el següent apartat.
+
+### Condicións de facturació
+
+Quan es genera una factura de manera **automàtica** o **manual**, l'**ERP**,
+fa una cerca de les línies extra relacionades amb el contracte amb les següents condicions:
+
+- Línies extra on la **data de la factura**(***date_invoice***) està **entre**
+la **data desde**(***date_from***) i la **data fins**(***date_to***), es
```
Comment (francescpuig7) [inline docs/facturacion/lineas_extra.md]: es a dir -> és a dir

File: docs/facturacion/lineas_extra.md [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+***Menú &rarr; Facturació &rarr; General &rarr; Línies Extra &rarr; Línies extra no facturades en contractes antics***
+
+
+![](_static/lineas_extra/extra_antics.png)
+
+### Veure línies extra asociades a un contracte
+
+
+Des de la vista ***Formulari*** d'un contracte, podem accedir a les línies
+extra del contracte a través del menú d'accions de la dreta ***Línies Extra***.
```
Comment (francescpuig7) [inline docs/facturacion/lineas_extra.md]: Potser millor parlar de menú de direccions o enllaços ja que están separats

File: docs/facturacion/lineas_extra.md [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+especificada en la línia extra i el que tocaria facturar.
+
+El preu de la línia de la factura, es calcularà en funció de la quantitat, el
+pendent a facturar i el preu per terme:
+
+- Si el pendent a facturar és inferior al preu per terme o unitat, s'afegirà
+a la línia de la factura tot el que queda per facturar.
+
+- Si la **data de la factura** (***date_invoice***) és **posterior o igual**
+    a la **data de finalització de la línia extra** (***date_to***), s'afegira
```
Comment (francescpuig7) [inline docs/facturacion/lineas_extra.md]: s'afegira -> s'afegirà
Revisar també traducció al castellà

File: docs/facturacion/lineas_extra.md [hunk 4]
```diff
@@ -0,0 +1,137 @@
...
+
+
+- En cas de que sigui una factura **rectificadora**
+(***tipo_rectificadora = R***), el **diari** (***journal_id***) de
+la **factura referenciada** (***factura.ref***),
+ha de coincidir amb algun dels **diaris** (***journal_ids***)
+de **línia extra**.
+
+
+**Mètodes**: ***fact_via_lectures()*** &rarr; ***compute_extra_lines()***
```
Comment (francescpuig7) [inline docs/facturacion/lineas_extra.md]: Revisar traducció al castellà

**Comment by francescpuig7** [comment]
Fix conflits please

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_docul_extra_lines/

Things to review:
- [ ] Typos in Catalan
- [ ] Typos in Spanish
- [ ] Links routes in Spanish
- [ ] Links work in Catalan
- [ ] Links work in Spanish
- [ ] Images in Spanish
- [ ] Page index shows all headers

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_docul_extra_lines/

Things to review:
- [ ] Typos in Catalan
- [ ] Typos in Spanish
- [ ] Links routes in Spanish
- [ ] Links work in Catalan
- [ ] Links work in Spanish
- [ ] Images in Spanish
- [ ] Page index shows all headers

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_docul_extra_lines/

Things to review:
- [ ] Typos in Catalan
- [ ] Typos in Spanish
- [ ] Links routes in Spanish
- [ ] Links work in Catalan
- [ ] Links work in Spanish
- [ ] Images in Spanish
- [ ] Page index shows all headers

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_docul_extra_lines/

Things to review:
- [ ] Typos in Catalan
- [ ] Typos in Spanish
- [ ] Links routes in Spanish
- [ ] Links work in Catalan
- [ ] Links work in Spanish
- [ ] Images in Spanish
- [ ] Page index shows all headers

**Comment by polsala** [comment]
> @polsala revisar traducció al cast: `se decir` i `añadira`

resolved :+1:

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_docul_extra_lines/

Things to review:
- [ ] Typos in Catalan
- [ ] Typos in Spanish
- [ ] Links routes in Spanish
- [ ] Links work in Catalan
- [ ] Links work in Spanish
- [ ] Images in Spanish
- [ ] Page index shows all headers

**Pull request merged**